tests/unit/multiclass/tree/RandomForest_unittest.cc

```diff
@@ -279,3 +283,69 @@ TEST(RandomForest,classify_non_nominal_test)
 	SG_UNREF(c);
 	SG_UNREF(eval);
 }
+
+TEST(RandomForest, test_scores)
```

Comment:
not sure I get what this test does.
Also it needs a better (self-explaining) name